test/src/main/java/org/apache/accumulo/test/TableOperationsIT.java

```diff
@@ -189,6 +190,18 @@ public void createTable() throws TableExistsException, AccumuloException,
     accumuloClient.tableOperations().delete(tableName);
   }
 
+  @Test(expected = IllegalArgumentException.class)
+  public void createTableWithTableNameLengthLimit()
+      throws AccumuloException, AccumuloSecurityException, TableExistsException {
+    StringBuilder tableNameBuilder = new StringBuilder();
+    for (int i = 0; i <= MAX_TABLE_NAME_LEN; i++) {
+      tableNameBuilder.append('a');
+    }
+    String tableName = tableNameBuilder.toString();
+    accumuloClient.tableOperations().create(tableName);
+    assertTrue(!accumuloClient.tableOperations().exists(tableName));
```

Comment:
When the previous line throws an exception, this line will not execute.

Reply:
Please see NamespacesIT comment.